src/authutil.ts

```diff
@@ -109,9 +113,9 @@ function writeFeedToFile(
   xml = xml.ele('packageSourceCredentials');
 
   sourceKeys.forEach(key => {
-    if (key.indexOf(' ') > -1) {
+    if (!isValidKey(key)) {
       throw new Error(
-        "This action currently can't handle source names with spaces. Remove the space from your repo's NuGet.config and try again."
+        "Source name can contain letters, numbers, and '-', '_', '.' symbols only, Please, fix source name in NuGet.config and try again."

```

Comment:
it should be a dot, not a comma.
```suggestion
        "Source name can contain letters, numbers, and '-', '_', '.' symbols only. Please, fix source name in NuGet.config and try again."

```